development/tools/jobs/kyma/apiserver_proxy_test_test.go

```diff
@@ -10,9 +10,9 @@ import (
 
 func TestApiserverProxyTestReleases(t *testing.T) {
 	// WHEN
-	unsupportedReleases := []string{"release-0.7"}
+	absentInSupportedReleases := []tester.SupportedRelease{tester.Release07}
```

Comment:
minor: unsupportedReleases is more simple than absentInSupportedReleases, so propose to not change that name